zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/annotation/PreFetchChangedData.java

```diff
@@ -22,8 +22,25 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
-
+/**
+ * An annotation used to prefetch changed data for
+ * {@link org.apache.helix.zookeeper.zkclient.IZkDataListener} in ZkClient.
+ * By default, prefetch is enabled: when ZkClient handles a data change event,
+ * ZkClient will read data and pass data object to
+ * {@link org.apache.helix.zookeeper.zkclient.IZkDataListener#handleDataChange(String, Object)}.
+ * By default, prefetch is disabled: ZkClient will not read data, so data object is passed as null.
```

Comment:
By default, prefetch is enabled or disabled? Line 28 and 31 are conflict.

Reply:
I don't know how I wrote two conflicting sentences @____@ Addressed.